sql/core/src/test/resources/sql-tests/results/postgreSQL/window_part3.sql.out

```diff
@@ -243,9 +243,8 @@ from t1 where f1 = f2
 -- !query schema
 struct<>
 -- !query output
-org.apache.spark.sql.AnalysisException
-cannot resolve '(PARTITION BY spark_catalog.default.t1.`f1` RANGE BETWEEN 1 PRECEDING AND 1 FOLLOWING)' due to data type mismatch: A range window frame cannot be used in an unordered window specification.; line 1 pos 24
```

Comment:
is there any result changing after this PR?

Reply:
> is there any result changing after this PR?

Yeah,  will add order spec then won't meet this problem